configure.ac

```
@@ -6251,6 +6248,9 @@ PY_STDLIB_MOD([fcntl],
   [], [$FCNTL_LIBS])
 PY_STDLIB_MOD([mmap],
   [], [test "$ac_cv_header_sys_mman_h" = "yes" -a "$ac_cv_header_sys_stat_h" = "yes"])
+PY_STDLIB_MOD([_posixshmem],
+  [], [test "$have_posix_shmem" = "yes"],
```

Would you prefer to have it this way? I'm not sure what's most correct here.

```suggestion
  [test "$have_posix_shmem" = "yes"], [],
```

---

The first check is for feature flags like ``--with-builtin-hashlib-hashes`` or ``--disable-test-modules``